libs/zzz/assets/src/index.ts

```
@@ -1,6 +1,54 @@
+  | 'normal'
+  | 'assist'
+  | 'skill'
+  | 'chain'
+  | 'core'
+type commonImagesKey = 'discBackdrop' | 'discDrive'
+const characterAssetCache = new Map<string, string | StaticImageData>()
+
+export function characterAsset(ck: CharacterKey, asset: characterAssetKey) {
+  const cacheKey = `${ck}_${asset}`
+  if (characterAssetCache.has(cacheKey))
+    return characterAssetCache.get(cacheKey)!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)